src/brs/db/sql/dialects/DatabaseInstanceSqlite.java

```diff
@@ -27,6 +27,19 @@ private String getJournalMode(){
     return "WAL";
   }
 
+  private String getSynchronousMode(){
```

Comment:
Only allow OFF, NORMAL, FULL as value - set NORMAL as default if not set correct